packages/interactive-monitor/src/index.ts

```
@@ -26,7 +26,7 @@ export const handler: SNSHandler = async (event) => {
 			`Expected exactly one record, but got ${event.Records.length}`,
 		);
 	} else {
-		const repo = event.Records[0]!.Sns.Message;
+		const repo = event.Records[0]!.Sns.Message.split('/')[1]!;
```

the non null assertion is a bit controversial but i think we should trust that repocop has sent what it's promised to send. If not, we should throw an error anyway